### DISCUSSION TITLE
Can I define and change suite name manually ?

### DISCUSSION THREAD

**Discussion by art-doppio** [reportportal/reportportal#1939] [Q&A]
I just start using ReportPortal with Robotframework and I would like to change suite name but I cannot find any document about this. ReportPortal will start with suite name according to how run command were used. In my case I want to run everything in `android` folder so my robot command ends with `./Testcases/android` and ReportPortal will start with Android.0 suite name and so on. Is there a way to override `Android.x` into something else ?
<img width="267" alt="image" src="https://user-images.githubusercontent.com/86337258/220600667-10613ee4-be93-40b9-9992-0f0b9f0a9e83.png">

https://user-images.githubusercontent.com/86337258/220601060-bec06c43-fa29-4a3d-9068-7e0924845925.mov

**Comment by art-doppio** [comment]
@DzmitryHumianiuk Could you help with this ?

**Reply by DzmitryHumianiuk** [reply]
it's better to ask @HardNorth here.

**Reply by art-doppio** [reply]
@HardNorth Could you please help with this
